pimd/pim_ifchannel.c

```diff
@@ -589,7 +599,7 @@ static int on_ifjoin_expiry_timer(struct thread *t)
 
   ch->t_ifjoin_expiry_timer = NULL;
 
-  ifjoin_to_noinfo(ch);
+  ifjoin_to_noinfo(ch, 1);
```

Comment:
Of course I realize that that 0/1  is used with other functions in these files. If this is more like a "standard" thing then we can live with it I guess.